### PR TITLE
chore: added mapping names to main README and capitalized vendor names

### PR DESCRIPTION
**Pull request by tlakomy**

### CODE REVIEW

File: schemas/ramp/v1/README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-| [Business.json](https://raw.githubusercontent.com/Stedi/registry/main/schemas/ramp/v1/Business.json)                                 | [![Map from this schema](/images/MapFromThisSchema.svg)](https://terminal.stedi.com/mappings/import?name=Mapping%20from%20ramp's%20Business%20schema&source_json_schema=https://raw.githubusercontent.com/Stedi/registry/main/schemas/ramp/v1/Business.json)                                 | [![Map to this schema](/images/MapToThisSchema.svg)](https://terminal.stedi.com/mappings/import?name=Mapping%20to%20ramp's%20Business%20schema&target_json_schema=https://raw.githubusercontent.com/Stedi/registry/main/schemas/ramp/v1/Business.json)                                 |
```
Comment (RafalWilinski) [inline schemas/ramp/v1/README.md:3]: This file is autogenerated but I don't see relevant code changes

**Review by RafalWilinski — CHANGES_REQUESTED** [review]
Please add relevant code changes in `generator.ts`

File: schemas/ramp/v1/README.md (hunk 1 above)
Comment (asennikov) [reply schemas/ramp/v1/README.md:3]: yeah, if we want to capitalize the provider name - we should do it here: https://github.com/Stedi/registry/blob/d5f916066490803de7b8b77ff706d16ef2d5940e/src/generator.ts#L129

**Comment by tlakomy** [comment]
Ah, right - thanks for the feedback, on it

**Pull request closed without merging**